<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>53</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QPushButton" name="pushButtonResultsFirstPhase">
     <property name="text">
      <string>Результаты 1 этапа</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonResultsSecondPhase">
     <property name="text">
      <string>Результаты 2 этапа</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonAllResults">
     <property name="text">
      <string>Все результаты</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
